README.md

```diff
@@ -1,20 +1,67 @@
 
-# To Do List using QT Designer and Sqlite <Python> 
+# To-Do List using QT Designer and Sqlite <Python> 
 
-This is an application for adding your daily tasks and checking the tasks you had done
+This is an application for adding your daily tasks and checking the tasks you have completed
 
+## Getting Started
 
+### Prerequisites
 
-## Authors
+- Python 3.7 or later
+- `virtualenv` package (optional, but recommended)
 
-- [@youssefkhalil](https://www.linkedin.com/in/youssef-khalil-4517641a2/)
+### 1. Cloning the Repository
+
+First, clone this repository to your local machine using git:
+```bash
+git clone https://github.com/youssefkhalil320/To-Do-List-application-with-calender-using-python-and-sqlite.git
+cd To-Do-List-application-with-calender-using-python-and-sqlite
+```
+### 2. Virtual Environment
+
+It's recommended to use a virtual environment to manage dependencies.
+```bash
+python -m venv venv
+```
+Activate the virtual library
+● Windows:
+```bash
+.\venv\Scripts\activate
+```
 
+● macOS and Linux:
+```bash
+source venv/bin/activate
+``` 
 
-## Dependencies
+### 3. Dependencies
 
+Once the virtual environment is activated, install the required dependencies in the requirements.txt file:
```

Comment:
no need for installing dependencies in this way.
use pip install -r requirements.txt instead

Reply:
Done